front/lib/api/assistant/generation.ts

```diff
@@ -330,12 +330,7 @@ export async function renderConversationForModelMultiActions({
         messages.unshift({
           role: "content_fragment",
           name: `inject_${m.contentType}`,
-          content:
-            `TITLE: ${m.title}\n` +
-            `TYPE: ${m.contentType}${
-              m.contentType === "file_attachment" ? " (user provided)" : ""
-            }\n` +
-            `CONTENT:\n${content}`,
+          content: `<file-attachment content-type="${m.contentType}" title="${m.title}">${content}</file-attachment>\n`,
```

Comment:
Let's add a`\n` after first tag and before last tag

Reply:
s/file-attachment/attachment/ since we have the content-type. Lets call it `type` instead of `content-type` since it's not really a content type